drawing_agent/app/agent.py

```
@@ -1,128 +1,103 @@
+    async def run(self, path: str, question: str, wait_time: Optional[int] = None,
+                  max_retries: Optional[int] = None, thread_id: Optional[str] = None,
+                  page: int = 0) -> Dict:
         start_time = time.time()
-        
-        wait_time = wait_time or (self.cfg.agent.get('wait_time', 4) if hasattr(self.cfg, 'agent') else 4)
-        max_retries = max_retries or (self.cfg.agent.get('max_retries', 4) if hasattr(self.cfg, 'agent') else 4)
-        thread_id = thread_id or (self.cfg.run.get('thread_id', 'default') if hasattr(self.cfg, 'run') else 'default')
-        
+        if thread_id is None:
+            thread_id = "default"
+        # Конфиг для LangGraph с thread_id
```

Ruff (0.15.12)

[warning] Comment contains ambiguous `с` (CYRILLIC SMALL LETTER ES). Did you mean `c` (LATIN SMALL LETTER C)?

(RUF003)